eZ/Publish/Core/MVC/Symfony/Controller/Content/QueryController.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
```

Comment:
I'd go for @license and @copyright alone, I don't really see the point of the first line anyway, and it seems that we don't really care about it.